lib/icinga/downtime.cpp

```diff
@@ -452,6 +456,19 @@ void Downtime::TriggerDowntime()
 	OnDowntimeTriggered(this);
 }
 
+void Downtime::SetCancellationInfo(const String& removedBy, double cancelTime, const MessageOrigin::Ptr &origin) {
+	Log(LogCritical, "Downtime") << "SetCancellationInfo(" << removedBy << ", " << cancelTime << ", " << (origin ? origin->ToString() : "null") <<")";
```

Comment:
Huh? LogCritical?

Reply:
That's printf debugging code that's of limited usefulness anyways, I'll remove it.